_docs/releasing/Discontinuing-a-project.md

```diff
@@ -0,0 +1,61 @@
+---
+title: Discontinuing a project
+author: OST
+date: 2020-20-05
+index: 3
+category: Releasing
+---
+
+**Overview of the different options for discontinuing a Zalando open source project, this process is to
+be used for projects which no longer have active maintainers, or projects which Zalando no longer have a motivation
+to maintain, usually due to lack of internal use.**
+
+
+### Summary
+
+- Projects which we have no motivation to maintain, we are obligated to gracefully discontinue
+- There are different options to ensure the project lives on
+- Projects can be archived, direct users to a 3rd party fork, or transfer ownership
+- The open source guild can help you make the right choice
+
+---
+
+## What projects should be discontinued?
+
+2 factors indicates if a project is ready for discontinuation. Either the project has no active maintainers, 
+and/or the project have no internal usage. While these 2 factors typically overlap, a project with internal
+usage and no maintainers need support to [find new internal maintainers]() rather than being discontinued.
```

Comment:
```suggestion
Two factors indicate whether a project is ready for being discontinued. Either the project has no active maintainers, 
and/or the project has no internal usage. While these two factors typically overlap, a project with internal
usage and no maintainers needs support to [find new internal maintainers]() rather than being discontinued.
```